source/renderer/app/components/status/DaedalusDiagnostics.scss

```diff
@@ -230,7 +224,11 @@
 =            SPECIFIC STYLES            =
 =======================================*/
 
-td.platformVersion {
+.layoutHeader.stateDirectoryPath {
+  width: 244px;
+}
+
+.layoutData.platformVersion {
   max-width: 435px;
```

Comment:
@DeeJayElly do we still need this max-width?

Reply:
Yes we need it in deed, parent element needs to have some kind of a limitation in order to know how to flex the child items.